src/init.h

```diff
@@ -73,4 +73,7 @@ bool AppInitMain(node::NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip
  */
 void SetupServerArgs(ArgsManager& argsman);
 
+/** Validates requirements to run the indexes and spawns each index initial sync thread */
+bool StartIndexes(node::NodeContext& node);
```

Comment:
commit 68b13a2b6c96fd061a31add58b8626c5bf1cdf3d:
I think that once it's decoupled from `BaseIndex::Init()`, the names `BaseIndex::Start()` and  `StartIndexes()` are misleading: It doesn't actually start the indexes, if `Init()` determines that an index is synced, the index is started right then (it begins to process validationinterface signals), and the later `StartIndexes()` does nothing, and could just as well be skipped. I'd prefer something like `IndexBackgroundSync()`.

Reply:
Done